source/manual/sentry.html.md

```diff
@@ -292,3 +292,7 @@ Sentry is used by several programmes in GDS, not just GOV.UK. A report,
 [GDS use of Sentry.io](https://docs.google.com/document/d/1yVa9iiu-DayGy-MtlrdXeV0E09CsetUenZJCmusLfLg/edit),
 covers this in more detail, including documenting some of the limitations of the
 setup.
+
+## Retention period
+
+[90 days](https://sentry.io/security/#data-retention).
```

Comment:
Do you know where this is defined? Is it in the Sentry UI or in code somewhere? I had a quick look in govuk-apppconfig but couldn't see it. Could be useful to link to that and then a separate link to the Sentry data retention policy?

Reply:
Yeah I also didn't have luck finding it either(!) so I've just linked to the standard Sentry policy for now.

Reply:
👍